coordinator/persistence-api/src/main/java/io/stargate/db/schema/SchemaHashable.java

```diff
@@ -49,11 +54,19 @@ static int hashCode(boolean b) {
 
   static int enumHashCode(Enum<?> object) {
     if (object == null) {
-      return 0;
+      return -1;
     }
     return object.name().hashCode();
   }
 
+  static int hashCode(Map<String, String> map) {
```

Comment:
Adding explicit methods to reduce need to call generic `hashCode(Object)` which is dangerous and should be removed eventually.